dropwizard-dependencies/pom.xml

```diff
@@ -477,6 +471,11 @@
                 <artifactId>mockito-junit-jupiter</artifactId>
                 <version>${mockito.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.mockito</groupId>
+                <artifactId>mockito-inline</artifactId>
+                <version>${mockito.version}</version>
+            </dependency>
```

Comment:
Do we need this new dependency? I don't see any changes in the usage of Mockito in our tests.